### DISCUSSION TITLE
Tag Overlays Should Increased And Be More Detailed

### DISCUSSION THREAD

**Discussion by HafizBeyy** [streetcomplete/StreetComplete#6046] [Feedback]
There are overlays; street lighting, surfaces, sidewaljs etc. They're not enough. For example, i saw a road with both lanes but it signed as one way road. i checked it with Organic Maps's router. i want to edit the tag and solve this issue. So i think overlays should increased. We can name one of them as "road". So this overlay makes volunteer edit every tag of road if he thinks anything is wrong.![Screenshot_20241219-153355_Street­Complete.png](https://github.com/user-attachments/assets/b364e08a-3a07-4bb4-99d8-09abf7357e95)
this is a both way road.
![Screenshot_20241219-153423_Organic Maps.png](https://github.com/user-attachments/assets/8a376cf6-285c-40bb-b4e0-32c9eb199c6d)
but the data is wrong organic maps's routing draws a weird direction.

**Comment by HolgerJeromin** [comment]
ref https://github.com/streetcomplete/StreetComplete/blob/master/OVERLAY_GUIDELINES.md

`🔎 One aspect: Similar to the atomic quests guideline, only one aspect should be displayed and be editable to the user.`

> For example, i saw a road with both lanes but it signed as one way road. i checked it with Organic Maps's router. i want to edit the tag and solve this issue. So i think overlays should increased. We can name one of them as "road".

Feel free to suggest adding specific overlays.

> So this overlay makes volunteer edit every tag of road if he thinks anything is wrong.

#5058
#471
The best guess is to use [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) "StreetComplete Expert Edition" fork of StreetComplete,
